.github/workflows/test-bionic.yml

```diff
@@ -0,0 +1,30 @@
+name: test-bionic
+on:
+  push:
+    paths-ignore:
+    - 'docs/**'
+  pull_request:
```

Comment:
@paddatrapper It should. That can be customized here.

Reply:
OK cool